continuous_integration/gpuci/axis.yaml

```diff
@@ -8,6 +8,6 @@ LINUX_VER:
 - ubuntu18.04
 
 RAPIDS_VER:
-- 21.08
+- "21.10"
```

Comment:
I wonder why this needs to be in quotes, but other arguments like `CUDA_VER` or `PYTHON_VER` don't

Reply:
I ended up putting quotes because without them, this was interpreted as a float and the trailing zero was cut off. Now that you mention that, I'm going to also put `PYTHON_VER` and `CUDA_VER` in quotes as well, so that it's clear that we mean for them to be interpreted as strings.

Reply:
Ah, I see. Nice catch!